Ardupilog.m

```diff
@@ -157,6 +157,8 @@ function delete(obj)
             % Construct the LineNo for the whole log
             LineNo_ndx_vec = sort(vertcat(obj.valid_msgheader_cell{:,2}));
             LineNo_vec = [1:length(LineNo_ndx_vec)]';
+            % Save the last LineNo as the number of messages in the log
+            obj.numMsgs = LineNo_vec(end);
```

Comment:
I was wondering about this: Should numMsgs contain the total number of msgs in the log, or just the amount of filtered messages?
What would be useful to you?